calico_versioned_docs/version-3.24/network-policy/istio/enforce-policy-istio.mdx

```diff
@@ -144,7 +144,7 @@ calicoctl create -f 30-policy.yaml
 :::note
 
 You can also
-[view the manifest in your browser](files/30-policy.yaml).
+[view the manifest in your browser](/files/30-policy.yaml).
```

Comment:
@ctauchen @girishshankaran yes, this is the forward slash that was missing. Without the forward slash, it's a relative path (to this mdx file), which, of course, does not exist.